lib/indexer/popularity_lookup.rb

```diff
@@ -16,37 +16,43 @@ def lookup_popularities(links)
             path_components: links,
           },
         },
-        _source: { includes: %w[rank_14] },
+        _source: { includes: %w[rank_14 vc_14] },
         sort: [
           { rank_14: { order: "asc" } },
         ],
         size: 10 * links.size,
       })
 
-      ranks = Hash.new(traffic_index_size)
+      default_rank = Hash.new(
+        rank: traffic_index_size,
+        view_count: 1,
+      )
 
-      results["hits"]["hits"].each do |hit|
+      ranks = results["hits"]["hits"].each_with_object(default_rank) do |hit, hsh|
         link = hit["_id"]
-        rank = Array(hit["_source"]["rank_14"]).first
+        rank = Array(hit.dig("_source", "rank_14")).first
+        view_count = hit.dig("_source", "vc_14") || 1
```

Comment:
Should this also have a fallback value for `rank`?  And if `rank` doesn't need one, why does `view_count`?

Reply:
Rank has one, sorry it's a bit convoluted. There's a `next if rank.nil?` in there, so we provide the default rank if there's no rank provided in the popularity index.

```ruby
      default_rank = Hash.new(
        rank: traffic_index_size,
        view_count: 1,
      )
```

We don't do the same for view_count because I want to preserve the behaviour we have live. So if rank is present but view_count isn't (I'm not sure why that'd happen), then `view_count` will still get a default value.